try-it.md

```diff
@@ -617,3 +553,49 @@ useful for troubleshooting to find out why a node did not deploy.
 | vendor_interface       | ipmitool                                                   |
 +-------------------------------------------------------------------------------------+
 ```
+
+
+### 2.9. Environment Variables
```

Comment:
I am more and more thinking that such a table with all possible variables should not be in a try-it page, but rather maybe in the actual Metal3-dev-env readme. Most of those options are unnecessary for people who just want to give it a go, and make the whole thing more complex to grab. Moving it to the end of the try-it is a good move. I would probably take it further by bringing this in the dev env repo. It will also be better to keep it in sync.

Reply:
Yes, I agree with you @maelk that it would be better to keep them in metal3-dev-env so that we don't miss updating documentation as well whenever we make related changes. 
Created [#376](https://github.com/metal3-io/metal3-dev-env/pull/376) to move those vars to dev-env